src/Maestro/MaestroApplication/ApplicationParameters/Prod.xml

```diff
@@ -5,9 +5,7 @@
   <Parameters>
     <Parameter Name="Environment" Value="Production" />
     <Parameter Name="WebCertName" Value="maestro-prod.westus2.cloudapp.azure.com" />
```

Comment:
Is the cert still needed?

Reply:
Missed that.
It shouldn't be, since we no longer use HTTPS